### PR TITLE
Language change dropdown bug#438/zubkov

### PR DESCRIPTION
**Pull request by olegz7**

### CODE REVIEW

File: src/app/app.component.ts [hunk 1]
```diff
@@ -1,14 +1,29 @@
...
   styleUrls: ['./app.component.scss']
 })
 
-export class AppComponent {
-  title = 'out-of-school';
-  constructor () {}
+export class AppComponent implements OnInit{
+  constructor() { }
+
+  MobileView: boolean = false;
```
Comment (dmitryy90) [inline src/app/app.component.ts:12]: this needs to be removed from header component and be passed as input parameter from app component into header conponent

**Review by dmitryy90 — CHANGES_REQUESTED** [review]

File: src/app/app.component.ts (hunk 1 above)
Comment (olegz7) [reply src/app/app.component.ts:12]: fixed, did same thing for sidenav component

Comment (dmitryy90) [reply src/app/app.component.ts:12]: i dont see any changes

**Review by dmitryy90 — CHANGES_REQUESTED** [review]

File: src/app/shared/components/sidenav/sidenav.component.ts [hunk 2]
```diff
@@ -59,6 +59,7 @@ export class SidenavComponent implements OnInit, OnDestroy{
   }
 
   ngOnInit(): void {
+    this.selectedLanguage = localStorage.getItem('ui-culture') || 'uk';
     this.user$.subscribe(user => this.user = user);
     this.isWindowMobile(window);
```
Comment (dmitryy90) [inline src/app/shared/components/sidenav/sidenav.component.ts:64]: you can remove handlers if you pass movileView param

**Review by dmitryy90 — CHANGES_REQUESTED** [review]

**Review by dmitryy90 — APPROVED** [review]

**Pull request merged**